### PR TITLE
Project 3: query multiple tables.

### PR DESCRIPTION
**Pull request by muhammedb8826**
### I have worked on the following : 

- Created owners table

create owners table

- Create species table

create species table

- Updated animals table:

     Removed column species
     Add column species_id  and owner_id which is a foreign key referencing species and owners table

add foreign key to animals

- Inserted data into the owners table and species table 

inserted data into owners

inserted data into species

- Update inserted animals so it includes the species_id value:
     If the name ends in "mon" it will be Digimon
     All other animals are Pokemon

includes species_id

Update inserted animals to include owner information (owner_id):

owner_id includes

- Queries (using JOIN) to answer the following questions:
> What animals belong to Melody Pond?

melody pond

> List of all animals that are pokemon (their type is Pokemon).

pokemone type

> List all owners and their animals, remember to include those that don't own any animal.

right join

> How many animals are there per species?

animals per species

> List all Digimon owned by Jennifer Orwell.

digimon owned orwell

> List all animals owned by Dean Winchester that haven't tried to escape.

dean winchester

> Who owns the most animals?

most animals

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -12,4 +12,22 @@ INSERT INTO animals(name,date_of_birth,weight_kg,neutered,escape_attempts) VALUE
...
+/* Project 3 */
+
+INSERT INTO owners (full_name,age) VALUES ('Sam Smith', 34);
+INSERT INTO owners (full_name,age) VALUES ('Jennifer Orwell', 19);
+INSERT INTO owners (full_name,age) VALUES ('Bob', 45);
+INSERT INTO owners (full_name,age) VALUES ('Melody Pond', 77);
+INSERT INTO owners (full_name,age) VALUES ('Dean Winchester', 14);
+INSERT INTO owners (full_name,age) VALUES ('Jodie Whittaker', 38);
+
+UPDATE animals SET species_id=2 WHERE name LIKE '%mon';
```
Comment (Shedrack-Sunday) [inline data.sql]: - [ ] In between these two, you're missing a major insertion based on project requirements.
- [ ] Insertion the following data into the species table:
`Pokemon
Digimon`

## Requirement:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @muhammedb8826       👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well so far in the project,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights   
- Good use SQL queries ✔️   

# Required Changes  🌵 

- [x] You made some impressive implementations based on the project requirements, but there are  more you need to adjust

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: data.sql (hunk 1 above)
Comment (muhammedb8826) [reply data.sql]: Hi Shedrack, Thank you for the feedback. I will add that to my data.sql file. However, I did that on the cmd. 😃

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @muhammedb8826 

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀

image


## Highlights 
- Proper use of GitHub PR  features ✔️ 
- Good use of SQL for queries ✔
- Kudus for taking heed from the last reviewer ✔️ 


## Optional Suggestions 💡
While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.
Cheers, and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**